setup.py

```diff
@@ -81,14 +77,12 @@ def create_pip_wheel():
         ],
         packages=["alphastats"],
         include_package_data=True,
-        entry_points={
-            "console_scripts": "alphastats=alphastats.cli:run",
-        },
+        entry_points={"console_scripts": "alphastats=alphastats.cli:run",},
         install_requires=requirements,
         extras_require=extra_requirements,
         python_requires=">=3.7,<4",
     )
 
 
 if __name__ == "__main__":
```

Comment:
I haven't seen a if __name__ == "__main__" in the setup.py before. Is there any reason this is needed or where it comes from?